### DISCUSSION TITLE
how to change FluentDatePicker to dd/mm/yyyy

### DISCUSSION THREAD

**Discussion by ifaizah** [microsoft/fluentui-blazor#2623] [Q&A] [answered]
`<div class="col-md-3">
    <FluentDatePicker Style="width:100%;" Label="Appointed Date" @bind-Value="@Content.appointed_date" Required />
</div>`

i have date picker, i want to change the format to dd/mm/yyyy but right now the format still d/m/yyyy
how i can change the format to dd/mm/yyyy to this form?

please help me, thank you.

**Comment by MarvinKlein1508** [comment, marked as answer]
Have you tried specifying a different culture? By default the component will display the value as the CurrentCulture from the user.

```razor
<FluentDatePicker Culture="System.Globalization.CultureInfo.InvariantCulture" />
```

**Reply by vnbaaij** [reply]
Not tested, but there is also the Blazor provided  `@bind:format="{your format string}"` you could try

**Reply by mahindrasuru** [reply]
Not working
